[32m2021-06-08 18:41:33[0m - dagster - [34mDEBUG[0m - camera_pipeline - 8189b2de-1254-4541-adda-038b8058723b - 6233 - create_geojson - LOGS_CAPTURED - Started capturing logs for solid: create_geojson.
[32m2021-06-08 18:41:33[0m - dagster - [34mDEBUG[0m - camera_pipeline - 8189b2de-1254-4541-adda-038b8058723b - 6233 - create_geojson - STEP_START - Started execution of step "create_geojson".
[32m2021-06-08 18:41:33[0m - dagster - [34mDEBUG[0m - camera_pipeline - 8189b2de-1254-4541-adda-038b8058723b - 6233 - create_geojson - LOADED_INPUT - Loaded input "kmls" using input manager "io_manager", from output "result" of step "correct_altitude_mode"
/home/pitanga/.local/share/virtualenvs/situated-views-XI40jPuu/lib/python3.8/site-packages/dagster/core/execution/plan/inputs.py:160: DtypeWarning: Columns (0,2,3,4,5,6,7,8,10,14,17,18,19,20,26,27,30,33,34,35,36,37,38) have mixed types.Specify dtype option on import or set low_memory=False.
  yield _load_input_with_input_manager(loader, load_input_context)
[32m2021-06-08 18:41:34[0m - dagster - [34mDEBUG[0m - camera_pipeline - 8189b2de-1254-4541-adda-038b8058723b - 6233 - create_geojson - LOADED_INPUT - Loaded input "metadata" using input manager "metadata_root"
[32m2021-06-08 18:41:35[0m - dagster - [34mDEBUG[0m - camera_pipeline - 8189b2de-1254-4541-adda-038b8058723b - 6233 - create_geojson - STEP_INPUT - Got input "kmls" of type "Any". (Type check passed).
[32m2021-06-08 18:41:35[0m - dagster - [34mDEBUG[0m - camera_pipeline - 8189b2de-1254-4541-adda-038b8058723b - 6233 - create_geojson - STEP_INPUT - Got input "metadata" of type "Any". (Type check passed).
[32m2021-06-08 18:41:40[0m - dagster - [34mERROR[0m - [31mcamera_pipeline - 8189b2de-1254-4541-adda-038b8058723b - 6233 - create_geojson - STEP_FAILURE - Execution of step "create_geojson" failed.

dagster.core.errors.DagsterExecutionStepExecutionError: Error occurred while executing solid "create_geojson"::

pyproj.exceptions.CRSError: CRS is empty or invalid: ''

Stack Trace:
  File "/home/pitanga/.local/share/virtualenvs/situated-views-XI40jPuu/lib/python3.8/site-packages/dagster/core/execution/plan/utils.py", line 42, in solid_execution_error_boundary
    yield
  File "/home/pitanga/.local/share/virtualenvs/situated-views-XI40jPuu/lib/python3.8/site-packages/dagster/utils/__init__.py", line 382, in iterate_with_context
    next_output = next(iterator)
  File "/home/pitanga/.local/share/virtualenvs/situated-views-XI40jPuu/lib/python3.8/site-packages/dagster/core/execution/plan/compute.py", line 126, in execute_core_compute
    for step_output in _yield_compute_results(step_context, inputs, compute_fn):
  File "/home/pitanga/.local/share/virtualenvs/situated-views-XI40jPuu/lib/python3.8/site-packages/dagster/core/execution/plan/compute.py", line 92, in _yield_compute_results
    user_event_generator = compute_fn(SolidExecutionContext(step_context), inputs)
  File "/home/pitanga/.local/share/virtualenvs/situated-views-XI40jPuu/lib/python3.8/site-packages/dagster/core/definitions/decorators/solid.py", line 329, in compute
    result = fn(context, **kwargs) if context_arg_provided else fn(**kwargs)
  File "/mnt/d/IMS/situated-views/bin/solids/camera.py", line 254, in create_geojson
    radius = get_radius(kml)
  File "/mnt/d/IMS/situated-views/bin/solids/camera.py", line 84, in get_radius
    depicted = reproject((float(lng), float(lat)))
  File "/mnt/d/IMS/situated-views/bin/solids/camera.py", line 27, in reproject
    rj = Proj('EPSG:32722', inverse= inverse)
  File "/home/pitanga/.local/share/virtualenvs/situated-views-XI40jPuu/lib/python3.8/site-packages/pyproj/proj.py", line 109, in __init__
    self.crs = CRS.from_user_input(projparams, **kwargs)
  File "/home/pitanga/.local/share/virtualenvs/situated-views-XI40jPuu/lib/python3.8/site-packages/pyproj/crs/crs.py", line 466, in from_user_input
    return CRS(value, **kwargs)
  File "/home/pitanga/.local/share/virtualenvs/situated-views-XI40jPuu/lib/python3.8/site-packages/pyproj/crs/crs.py", line 305, in __init__
    projkwargs = _prepare_from_dict(kwargs, allow_json=False)
  File "/home/pitanga/.local/share/virtualenvs/situated-views-XI40jPuu/lib/python3.8/site-packages/pyproj/crs/crs.py", line 68, in _prepare_from_dict
    return _prepare_from_string(" ".join(pjargs))
  File "/home/pitanga/.local/share/virtualenvs/situated-views-XI40jPuu/lib/python3.8/site-packages/pyproj/crs/crs.py", line 73, in _prepare_from_string
    raise CRSError(f"CRS is empty or invalid: {in_crs_string!r}")
[0m
